bricks/advanced/src/next-table/interface.ts

```diff
@@ -1,5 +1,39 @@
-import type { ColumnType } from "antd/es/table";
+import type { ColumnType, TablePaginationConfig } from "antd/es/table";
+import type { UseSingleBrickConf } from "@next-core/types";
 
 export type RecordType = Record<string, any>;
 
-export interface Column extends ColumnType<RecordType> {}
+export interface Column extends ColumnType<RecordType> {
+  /** 表头自定义展示构件 */
+  headerBrick?: {
+    useBrick: UseSingleBrickConf | UseSingleBrickConf[];
```

Comment:
`"@next-core/types` 里有个类型

```suggestion
    useBrick: UseBrickConf;
```